src/tqec/templates/scalable/rectangle.py

```diff
@@ -4,19 +4,37 @@
 
 
 class ScalableRectangle(ScalableTemplate):
-    def __init__(self, width: int, height: int) -> None:
+    """Rectangle with scalable width **or** height."""
```

Comment:
I would remove that comment and include it as the first line of the `__init__` docstring. 
In general, I find it more meaningful to document the class behaviour in the `__init__` method, as it helps in documenting everything (i.e., also the `__init__` parameters) **and** avoid meaningless docstrings in the `__init__` that simply refer to the main class documentation. Also, as far as I am aware, most of the documentation-generation tools allow to replace the class documentation by the `__init__` one.